pkg/controller/manager/types_old.go

```diff
@@ -177,3 +186,60 @@ func unescapeFilters(strs []string) []string {
 	}
 	return strs
 }
+
+// parseStartDate parses the supplied string as a strfmt.DateTime.
+func parseStartDate(value string) (strfmt.DateTime, error) {
+	now := timeutc.Now()
+
+	if value == "now" {
+		return strfmt.DateTime(now.Add(nowSafety)), nil
+	}
+
+	if strings.HasPrefix(value, "now") {
+		d, err := duration.ParseDuration(value[3:])
+		if err != nil {
+			return strfmt.DateTime{}, err
+		}
+		return strfmt.DateTime(now.Add(d.Duration())), nil
+	}
+
+	// No more heuristics, assume the user passed a date formatted string
+	t, err := timeutc.Parse(time.RFC3339, value)
+	if err != nil {
+		return strfmt.DateTime(t), err
+	}
+	return strfmt.DateTime(t), nil
+}
+
+var (
+	byteCountRe          = regexp.MustCompile(`([0-9]+(?:\.[0-9]+)?)(B|[KMGTPE]iB)`)
+	byteCountReValueIdx  = 1
+	byteCountReSuffixIdx = 2
+)
+
+// parseByteCount returns byte count parsed from input string.
+// This is opposite of StringByteCount function.
+func parseByteCount(s string) (int64, error) {
```

Comment:
ditto

Reply:
again, moved away from manager https://github.com/scylladb/scylla-manager/commit/edbbf605148b9f765cc79e72def5e69c69683377, still part of our API though